config/database.yml

```diff
@@ -20,10 +20,14 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: "20062000"
```

Comment:
- [ ] Please make sure to not leave any credentials info in your published repo for security matters. Also, I recommend that you put the username and password under the `default` property in the file, and it will be applied automatically in the `development`, `test`, and `production`.